content/tutorials/managing-config-and-secrets/index.md

```diff
@@ -0,0 +1,194 @@
+summary: |
+    In Pulumi, an environment is a collection of values intended to capture the configuration values needed to work with a particular environment. These can be raw values like server names, environment types, region names and so on. They can also be sensitive values such as database passwords or service tokens.
+    
+    In this tutorial, we'll demonstrate how to create and utilize configuration and secret values in Pulumi.
+
+# A list of three to five things the reader will have learned by the end of the tutorial.
+youll_learn:
+    - How to create a raw configuration value
+    - How to create a secret configuration value
+    - How to view configuration details in the CLI
+    - How to access configuration values from within Pulumi program code
+
+# A list of tutorial prerequisites. Markdown is fine. Keep it simple; no need to be exhaustive here.
+prereqs:
+    - The [Pulumi CLI](/docs/install/)
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)
+    - One of Pulumi’s [supported language runtimes](/docs/languages-sdks/) installed
+
+# The estimated time, in minutes, for new users to complete the topic.
+estimated_time: 10
+
+# # An optional list of collections this tutorial should be belong to. Collections are defined in data/tutorials/collections.yaml.
+# collections:
+#     - some-non-existent-collection
+---
+
+## Create a new project
+
+To start, login to the [Pulumi CLI](/docs/cli/commands/pulumi_login/). Next, create a new project by running the `pulumi new <language>` command, making sure to replace `<language>` with the supported language of your choice:
+
+```bash
+# example using python
+$ pulumi new python
+This command will walk you through creating a new Pulumi project.
+
+Enter a value or leave blank to accept the (default), and press <ENTER>.
+Press ^C at any time to quit.
+
+project name (pulumi-dev):  
+project description (A minimal Python Pulumi program):  
+Created project 'pulumi-dev'
+
+Please enter your desired stack name.
+To create a stack in an organization, use the format <org-name>/<stack-name> (e.g. `acmecorp/dev`).
+stack name (dev): pulumi/dev
+Created stack 'dev'
+
+Installing dependencies...
+
+Creating virtual environment...
+Finished creating virtual environment
+Updating pip, setuptools, and wheel in virtual environment...
+...
+...
+Finished installing dependencies
+
+Your new project is ready to go!
+
+To perform an initial deployment, run `pulumi up`
+```
+
+This will create simple Pulumi program without any resources or configuration details.
+
+## Create configuration values
+
+In a Pulumi project, you can locally [store and retrieve configuration values](/docs/concepts/config/) using the `pulumi config set <key> [value]` command. To demonstrate, run the following command to create a configuration value with a key of `myEnvironment` and a value of `development`:
```

Comment:
See previous comment. I think we should use a different example configuration value other than the overloaded term "environment".

Reply:
On this one I will keep as is. The example program code being leveraged is also being used in another tutorial and I would basically have to duplicate this code just to change `myEnvironment` to something else. I have however updated the intro paragraph to make this less confusing.